cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/util/AbstractAppFabricHttpHandler.java

```diff
@@ -93,7 +93,7 @@
   /**
    * Class to represent status of programs.
    */
-  protected static final class AppFabricServiceStatus {
+  public static final class AppFabricServiceStatus {
```

Comment:
Why? If it is needed to be used across multiple classes, turn it into a top level class (which seems unlikely, since this class uses HttpResponseStatus inside, which is very http handler specific).